src/server/api/boards/boards.js

```diff
@@ -100,6 +100,47 @@ function addCardToColumn(board, columnId, cardName) {
   });
 }
 
+function validateColumnId(board, columnId, newColumnId) {
```

Comment:
i assume `columnId` is the column from where we move the card and `newColumnId` is where we want to move.

`fromColumnId` and `toColumnId` might be more descriptive.